tests/end2end/cypress/integration/print-ghaction.js

```diff
@@ -210,4 +210,12 @@ describe('Print', function () {
     //     })
     // })
 
+
+    it('should get atlas print', function () {
+        cy.get('#layer-quartiers button.btn.checkbox').click()
+        // Click quartier
+        cy.get('#map').click(480,340)
```

Comment:
```suggestion
        cy.mapClick(480,340)
```
Let's switch to this helper, make it easier one day if we change coordinates.

Reply:
I have done a copy/paste, I'll fix the other tests!